src/app/features/ledger/flows/request-bitcoin-keys/ledger-request-bitcoin-keys.tsx

```diff
@@ -45,11 +48,13 @@ function LedgerRequestBitcoinKeys() {
           onRequestKey(index) {
             if (index <= 4) {
               ledgerNavigate.toDeviceBusyStep(
-                `Requesting Bitcoin Native Segwit address (${index + 1}…5)`
+                `Requesting Bitcoin Native Segwit address (${index + 1}…${defaultNumberOfKeysToPullFromLedgerDevice})`
```

Comment:
@kyranjamie : Something looks funny here. When I test this:

- We show `Requesting Bitcoin Native Segwit address` `X-10` but we only load 5 
- Then we show `Requesting Bitcoin Taproot address` `X-10` and again go up to just 5
- Finally we show `Requesting Bitcoin Taproot address` `X-10` and count from 1 - 10

I guess there is some UI bug here and that we should be loading 10 of each address type? It seems like we incorrectly loop twice for the native segwit addresses. 

I can investigate it 




https://github.com/leather-wallet/extension/assets/2938440/b00f2d07-971c-4904-a55f-5844a04895e4

Reply:
I figured this out and fixed it [here](https://github.com/leather-wallet/extension/pull/5353/commits/910a39d88f3ced20296203ec6709cc1e8eecc47e)